src/test/java/no/unit/nva/model/contexttypes/BookTest.java

```diff
@@ -148,7 +126,7 @@ void bookThrowsInvalidIsbnExceptionWhenIsbnIsInvalid(String seriesTitle,
                                                          String peerReviewed,
                                                          String isbnList) {
         List<String> convertedIsbnList = convertIsbnStringToList(isbnList);
-        ArrayList<String> expectedIsbnList = isNull(convertedIsbnList) ? null : new ArrayList<>(convertedIsbnList);
+        ArrayList<String> expectedIsbnList = new ArrayList<>(convertedIsbnList);
```

Comment:
You do not expect this value anywhere, it seems more of a an `invalidIsbnList` or a `listWithInvalidIsbn`

Reply:
In general the "expected" variables here are not expected and it is confusing for the reader